tests/tracking/test_log_figure.py

```diff
@@ -45,6 +45,37 @@ def test_log_figure_plotly(subdir):
         assert os.listdir(run_artifact_dir) == [filename]
 
 
+@pytest.mark.parametrize("extension", ['png', 'jpeg', 'webp', 'svg', 'pdf'])
+def test_log_figure_plotly_image(extension):
+    from plotly import graph_objects as go
+    
+    subdir = "."
+    filename = f"figure.{extension}"
+    artifact_file = posixpath.join(subdir, filename)
+
+    fig = go.Figure(go.Scatter(x=[0, 1], y=[2, 3]))
+
+    with mlflow.start_run():
+        mlflow.log_figure(fig, artifact_file)
+
+        artifact_path = None if subdir is None else posixpath.normpath(subdir)
+        artifact_uri = mlflow.get_artifact_uri(artifact_path)
+        run_artifact_dir = local_file_uri_to_path(artifact_uri)
+        assert os.listdir(run_artifact_dir) == [filename]     
+        
+
+def test_log_figure_raises_error_for_unsupported_file_extension():
```

Comment:
```suggestion
@pytest.mark.parametrize("extension", ['', '.py'])
def test_log_figure_raises_error_for_unsupported_file_extension(extension):
```

Can we also parametrize this test?